content/review.rst

```diff
@@ -256,7 +264,7 @@ Reviewer's check list - Any Role
 
 * Is there a release notes entry for changes?
 
-* Is there documentation for changes? 
+* Is there documentation for changes?
```

Comment:
Yes, corrected, makes sense. I reordered a bit that section so it is better now too.